routes/eventRoutes.js

```diff
@@ -11,6 +11,8 @@ router.get("/", async (req, res, next) => {
 	try {
 		let data = await db.findAll("Events");
 
+		sendEmail("rane@blasph.me", "test", "test");
```

Comment:
This is a bit of debugging code that I plan on taking out, if you would like to remove it here that works too

Reply:
> Approved, if you take out the debugging code let me know, otherwise I will submit a fix once this is merged.

@RaneWallin - thanks Rane. I'll remove your debugging code from my branch before merging so we're in sync.